py/farm_ng/canbus/packet.py

```diff
@@ -491,3 +492,133 @@ def is_button_pressed(self, button: PendantButtons) -> bool:
 
     def __str__(self):
         return "x {:0.3f} y {:0.3f} buttons {}".format(self.x, self.y, self.buttons)
+
+
+class BugDispenserRpdo3(Packet):
+    """Bug dispenser rate in m/drop (request) sent to the Amiga dashboard."""
+
+    cob_id = 0x400
+
+    def __init__(self, rate1=0, rate2=0, rate3=0):
+        self.rate1 = rate1
+        self.rate2 = rate2
+        self.rate3 = rate3
```

Comment:
If node ids are using 0 indexing (0 - 2), then these should too for consistency
```suggestion
        self.rate0 = rate0
        self.rate1 = rate1
        self.rate2 = rate2
```